android/app/src/main/res/values/dimens.xml

```diff
@@ -40,7 +42,7 @@
     <dimen name="heading_text_size">20sp</dimen>
     <dimen name="text_size_expanded_title">22sp</dimen>
     <dimen name="text_size_extra_large">18sp</dimen>
-    <dimen name="text_size_large">18sp</dimen>
+    <dimen name="text_size_large">16sp</dimen>
```

Comment:
@arp95 you changed this in you PR [#1755](https://github.com/fossasia/open-event-android/pull/1755) . Please suggest what should be done. Technically it will revert your change.